### ISSUE TITLE
Show subsets of datasets found by searching in the datasets panel

### ISSUE THREAD

**Issue by Dmitrii-Romanov** [epam/NGB#269]
Would be very useful if the search in the datasets panel should be able to show all subsets for datasets that are the result of the search even if subsets doesn't contain a search substring in their names.

**Comment by malperovich** [comment]
Дима,

Ты мог бы для меня пояснить – что имеется в виду?
Из текста я не понял – показывать в рез-тах поиска даже те датасеты, в именах файлов которых отсутствует строка поиска?
Как они тогда попадут в результаты поиска?..

**Comment by Dmitrii-Romanov** [comment]
Да. Именно так. Датасет имеет свое имя (имя проекта, например). Его дочерние датасеты могут иметь другие имена, не попадающие под критерии поиска, но их нужно видеть, ибо пользователю нужно работать с проектом.
Например, при поиске “fruit” вылезает только имя родительского датасета, а под ним пусто – смысла в этой ситуации немного
http://ngb.opensource.epam.com/catgenome/#/

Формировать имена, которые включают в себя названия всех вышестоящих датасетов – требует дополнительных усилий, не очень удобно даже с т.з. отображения в датасет панели, небольшая опечатка приведет к неотображению сабсета.



Dmitrii Romanov
Project Manager

Office: +7 812 611 10 94 x 27400<tel:+7%20812%20611%2010%2094;ext=27400>   Email: dmitrii_romanov@epam.com<mailto:dmitrii_romanov@epam.com>
Saint-Petersburg, Russia   epam.com<http://www.epam.com>

CONFIDENTIALITY CAUTION AND DISCLAIMER
This message is intended only for the use of the individual(s) or entity(ies) to which it is addressed and contains information that is legally privileged and confidential. If you are not the intended recipient, or the person responsible for delivering the message to the intended recipient, you are hereby notified that any dissemination, distribution or copying of this communication is strictly prohibited. All unintended recipients are obliged to delete this message and destroy any printed copies.
